### ISSUE TITLE
eBay Marketplace Account Deletion/Closure Notifications Workflow

### ISSUE THREAD

**Issue by doverradio** [hendt/ebay-api#135]
Hello, I recently got notified I had not yet complied with the "[eBay Marketplace Account Deletion/Closure Notifications Workflow](https://developer.ebay.com/marketplace-account-deletion)".  I have been trying to generate a proper endpoint for this task.  

Do you plan to add this workflow to `ebay-api`?

**Comment by doverradio** [comment]
Nevermind, its more involved than just the endpoint.  Had to configure also directly on the developer account settings.

**Comment by dantio** [comment]
Could you elaborate more how You solved it? It might be a good idea to put it here in the docs.
Did You used the notification Api?

**Comment by doverradio** [comment]
I had to set up a get endpoint to receive the request coming from eBay if a user does Account Deletion request.  I didn't know but I guess about 1 year ago this policy got implemented and if you don't follow, they can terminate your developer account.  So, after setting up the endpoint, I get the request then had to use their node sdk to obtain their token.  Then had to respond with the token and additional steps to do base64 encoded string of my creds plus token.  Similar to how you OAuth already.   Get code, respond, get token, etc.
